knowledge-base/grid-remove-group-indent.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Removing Group Indent in Grid for Blazor
+description: This article demonstrates how to remove the group indent in the Grid for Blazor by overriding the default theme styles.
+type: how-to
+page_title: How to Remove Group Indentation in Blazor Grid
+slug: grid-kb-remove-group-indent
+tags: grid, blazor, grouping, indentation
+res_type: kb
+ticketid: 1684110
+---
+
+## Description
+
+This knowledge base article also answers the following questions:
+
+- Is there a way to remove the group indent and adjust the spacing without compromising the Grid behavior?
+- How can I customize the grouping appearance in a Blazor Grid?
+- How to hide the group indent and group header icons in a Blazor Grid?
+- What is the best way to adjust the appearance of grouped headers in a Blazor Grid?
```

Comment:
The last question is not clear enough as there are multiple ways to customize the headers. Let's be more specific here and list that the example shows how to hide the expand/collapse icon for the group rather than general customization of the group headers.